src/components/elements/background-image.js

```diff
@@ -14,12 +14,12 @@ const StyledBackground = styled(BackgroundImage)`
     }
 `
 
-export const Background = ({ children, data, style, dark, ...props }) => {
+export const Background = ({ children, data, fluid, style, dark, ...props }) => {
     const image = getImage(data)
     const bgImage = convertToBgImage(image)
 
     return (
-        <StyledBackground Tag="div" style={style} dark={dark} {...bgImage} {...props}>
+        <StyledBackground Tag="div" style={style} dark={dark} fluid={fluid} {...bgImage} {...props}>
```

Comment:
You shouldn't need to add this. Using the original `data` prop, passing it like usual will work.

Reply:
already tried this but it's not working if we try adding linear-gradient..Im following this guideline https://www.gatsbyjs.com/plugins/gatsby-background-image/#how-to-use-with-multiple-images..